.github/workflows/testing.yml

```diff
@@ -17,10 +17,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: Setup temporary directory
```

Comment:
It could have been working fine before, I just created the path to the tmp directory wrong:
https://github.com/limebit/medmodels/pull/59/files#diff-7dbc9b460fc40c46edb70b6ef6a6e0632f9115e422bad06fc538b15c2aa8a195R472

It created a path to `/tmpmedrecord_test` instead of `tmp/medrecord_test` which caused the `Permission denied` error